benchmarks/benchmarks/_utils.py

```diff
@@ -43,8 +63,11 @@ def _pbmc3k() -> AnnData:
     sc.pp.calculate_qc_metrics(
         adata, qc_vars=["mt"], percent_top=None, log1p=False, inplace=True
     )
+    assert isinstance(adata.X, sparse.csr_matrix)
     adata.layers["counts"] = adata.X.astype(np.int32, copy=True)
+    adata.layers["counts-off-axis"] = adata.layers["counts"].tocsc()
     sc.pp.log1p(adata)
+    adata.layers["off-axis"] = adata.X.tocsc()
```

Comment:
same here, this just differs based on `X` being dense or sparse

Reply:
I mean we could make a helper function for that.